src/index.js

```diff
@@ -628,7 +629,12 @@ module.exports = async (request, response, config = {}, methods = {}) => {
 	}
 
 	if (!stats) {
-		return sendError(absolutePath, response, acceptsJSON, current, handlers, config, {
+		// allow for custom 404 handling
+		const headers = await getHeaders(config.headers, current, absolutePath, stats);
+		Object.keys(headers).forEach(key => {
+			response.setHeader(key, headers[key]);
```

Comment:
I'm not 100% sure of this, but it does seem like it's a missing feature in the code…maybe. I don't think it's required by any measure. 

Hmm, maybe I should just remove it entirely in that case…